programs/ssl/ssl_client2.c

```diff
@@ -3292,6 +3309,13 @@ int main( int argc, char *argv[] )
     fflush( stdout ); getchar();
 #endif
 
+#if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C)
+#if defined(MBEDTLS_MEMORY_DEBUG)
+    mbedtls_memory_buffer_alloc_status();
+#endif
+    mbedtls_memory_buffer_alloc_free();
+#endif
```

Comment:
For consistency with ssl_server2, we should print the status and call free before the "Press Enter" and `getchar()` block.

Reply:
done